### PR TITLE
Map Synchronization

### PR DESCRIPTION
**Pull request by bhdrks78**
Synchronization of maps in Client and QM

### CODE REVIEW

File: package/Maps/Yuri's Revenge/2_Indonesia_navalwar.map
Comment (alexp8) [inline package/Maps/Yuri's Revenge/2_Indonesia_navalwar.map:1]: Can you keep map names lower case? Prevents any case clashing in the future @bhdrks78

File: package/Maps/Yuri's Revenge/4_treetop_hills.map
Comment (alexp8) [inline package/Maps/Yuri's Revenge/4_treetop_hills.map:1]: is this map file 4 MB ?

File: package/Maps/Yuri's Revenge/4_treetop_hills.map [hunk 1]
```diff
@@ -0,0 +1,3696 @@
...
+Waypoint4=289,101
+Waypoint5=0,0
+Waypoint6=0,0
+Waypoint7=0,0
+Waypoint8=0,0
+NumberStartingPoints=4
+
+[GAPOWR]
+Sight=7
+BuildTimeMultiplier=0.75
```
Comment (alexp8) [inline package/Maps/Yuri's Revenge/4_treetop_hills.map:22]: hard coded allied power plant buff?

File: package/Maps/Yuri's Revenge/4_treetop_hills.map [hunk 2]
```diff
@@ -0,0 +1,3696 @@
...
+
+[INYELWLAMP]
+LightRedTint=0.13
+LightBlueTint=0.15
+LightGreenTint=0.17
+LightIntensity=0.06
+LightVisibility=3500
+
+[NAPOWR]
+Sight=7
```
Comment (alexp8) [inline package/Maps/Yuri's Revenge/4_treetop_hills.map:74]: why does soviet tesla reactor have increased sight?

File: package/INI/MPMaps.ini [hunk 3]
```diff
@@ -8179,4 +8164,84 @@ Waypoint3=128180
 Waypoint4=178130
 Waypoint5=177106
 Waypoint6=127057
-Waypoint7=104056
\ No newline at end of file
+Waypoint7=104056
+
+[Maps\Yuri's Revenge\2_Indonesia_navalwar]
+Author=Westwood & Instinct Edit
```
Comment (alexp8) [inline package/INI/MPMaps.ini:8170]: Indonesia is westwood?

**Review by alexp8 — CHANGES_REQUESTED** [review]

File: package/Maps/Yuri's Revenge/2_Indonesia_navalwar.map
Comment (bhdrks78) [reply package/Maps/Yuri's Revenge/2_Indonesia_navalwar.map:1]: overlooked

File: package/Maps/Yuri's Revenge/4_treetop_hills.map (hunk 1 above)
Comment (bhdrks78) [reply package/Maps/Yuri's Revenge/4_treetop_hills.map:22]: Yes, he used this code to make it easier to build on some maps.

File: package/INI/MPMaps.ini (hunk 3 above)
Comment (bhdrks78) [reply package/INI/MPMaps.ini:8170]: There was no author in the map and I found the name of this author in the Chinese maps channel on discord.

File: package/Maps/Yuri's Revenge/4_treetop_hills.map (hunk 1 above)
Comment (alexp8) [reply package/Maps/Yuri's Revenge/4_treetop_hills.map:22]: let's not hardcode the power plant buff into maps

**Review by alexp8 — APPROVED** [review]

**Pull request merged**